sources/trophy.move

```diff
@@ -0,0 +1,310 @@
+#[allow(lint(share_owned, self_transfer))]
+module collection::trophy {
```

Comment:
There is no mapping or reference to fraction ID here? how are we storing fraction ID and minting 1 NFT corresponding to 1 unique fraction ID?